docs/release-notes/8.4.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[release-notes-header-8.4.0]]
 == 8.4
 
+[discrete]
+[[release-notes-8.4.3]]
+=== 8.4.3
+
+[discrete]
+[[bug-fixes-8.4.3]]
+==== Bug fixes and enhancements
+* Aligns the delete icon in the Add Rule Exception flyout ({pull}141365[#141365]).
+* Aligns the warning message title on the Rule details page with the warning icon  ({pull}140719[#140719]).
```

Comment:
@nastasha-solomon could you please also remove this one from the [8.4.2 release notes](https://security-docs_2502.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.4.0.html#bug-fixes-8.4.2)?

Reply:
```suggestion
```

Reply:
Reverting this change and removing the PR from the 8.4.2 release notes.